src/types.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+export const MovieCardType = {
+  title: PropTypes.string.isRequired,
+  imgUrl: PropTypes.string.isRequired,
+  imdbUrl: PropTypes.string.isRequired,
+  imdbId: PropTypes.string.isRequired,
+  description: PropTypes.string,
+};
+
+export const MoviesListTypes = {
```

Comment:
Name all the types in a similar way, e.g. 'MoviesListType', not 'MoviesListTypes'